### PR TITLE
Migrating MySQL to community.mysql

### PR DESCRIPTION
**Pull request by bmildren**
##### SUMMARY
Ready to merge once community.mysql has been released.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
mysql_*

### CODE REVIEW

**Comment by felixfontein** [comment]
Two files are missing:
```
changelogs/fragments/618-mysql_user_add_missed_privileges.yml
changelogs/fragments/609-mysql_user_fix_overriding_password_to_the_same.yml
```
Also, a lot of `version_added`s needs to be adjusted in the new collection. (Since these modules were actively worked on after the migration to community.general.)

**Comment by bmildren** [comment]
Thanks @felixfontein! 🙂   I've removed the changelog fragments, and have made a note about the `version_added`s and will adjust those accordingly 👍

File: meta/runtime.yml [hunk 1]
```diff
@@ -627,14 +657,25 @@ plugin_routing:
...
+      deprecation:
+        removal_version: 2.0.0
+        warning_text: The mysql docs_fragment has been moved to the community.mysql collection.
+      redirect: community.mysql.mysql
     proxysql:
       deprecation:
         removal_version: 2.0.0
         warning_text: The proxysql docs_fragment has been moved to the community.proxysql collection.
       redirect: community.proxysql.proxysql
+
```
Comment (felixfontein) [inline meta/runtime.yml:670]:
```suggestion
```

File: tests/sanity/ignore-2.9.txt [hunk 2]
```diff
@@ -500,9 +500,6 @@ plugins/modules/database/misc/riak.py validate-modules:doc-default-does-not-matc
 plugins/modules/database/misc/riak.py validate-modules:doc-missing-type
 plugins/modules/database/misc/riak.py validate-modules:parameter-type-not-in-doc
 plugins/modules/database/mssql/mssql_db.py validate-modules:doc-missing-type
```
Comment (felixfontein) [inline tests/sanity/ignore-2.9.txt:502]:
```suggestion
```

File: tests/sanity/ignore-2.11.txt [hunk 3]
```diff
@@ -631,14 +631,6 @@ plugins/modules/database/misc/riak.py validate-modules:doc-missing-type
 plugins/modules/database/misc/riak.py validate-modules:parameter-type-not-in-doc
 plugins/modules/database/mssql/mssql_db.py validate-modules:doc-missing-type
 plugins/modules/database/mssql/mssql_db.py validate-modules:doc-required-mismatch
```
Comment (felixfontein) [inline tests/sanity/ignore-2.11.txt:633]:
```suggestion
```

File: tests/sanity/ignore-2.10.txt [hunk 4]
```diff
@@ -631,14 +631,6 @@ plugins/modules/database/misc/riak.py validate-modules:doc-missing-type
 plugins/modules/database/misc/riak.py validate-modules:parameter-type-not-in-doc
 plugins/modules/database/mssql/mssql_db.py validate-modules:doc-missing-type
 plugins/modules/database/mssql/mssql_db.py validate-modules:doc-required-mismatch
```
Comment (felixfontein) [inline tests/sanity/ignore-2.10.txt:633]:
```suggestion
```

**Comment by felixfontein** [comment]
Hmm, that was too much...

**Comment by felixfontein** [comment]
Will merge once sanity checks pass.

**Comment by codebymikey** [comment]
New repository is available here: https://github.com/ansible-collections/community.mysql

**Pull request merged**